homeassistant/generated/dhcp.py

```diff
@@ -66,6 +66,11 @@
         "domain": "flume",
         "hostname": "flume-gw-*"
     },
+    {
+        "domain": "goalzero",
+        "hostname": "yeti*",
+        "macaddress": "A4CF12*"
```

Comment:
"A4CF12" is one of the ranges registered on Espressif, is it documented that goalzero devices will always use this range, or is it better to ommit it?

Reply:
OK, but espressif have multiple registered MAC address ranges, not just this one.